### PR TITLE
add code of conduct

### LINKED ISSUE #9: Add 'Code of Conduct' page
this is a must and should be easily findable from anywhere on the website, probably should have a link in home and in footer.

https://jupytercon.com/codeofconduct/

### PR DESCRIPTION
**Pull request by OriolAbril**
Add code of conduct to website. fixes #9

I have written in markdown which is not rendered correctly in the website even though it's jekyll powered. Once the content is final, we can work on formatting.

tagging @twiecki @sidravi1 @canyon289 @AlexAndorra

### CODE REVIEW

File: _2017_pages/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  incident is ongoing, and/or if this is part of an ongoing pattern of behavior
+* Names and contact info, if possible, of anyone else who witnessed or was involved in
+  this incident. (Did anyone else observe the incident?)
+* Any other relevant information you believe we should have
+
+_The Diversity committee will attempt to gather and write down the above information from
+anyone making a verbal report. Recording the details in writing is exceedingly important
+in order for us to effectively respond to reports. If event staff write down a report
+taken verbally, then the person making the report will be asked to review the written
+report for accuracy._
```
Comment (OriolAbril) [inline _2017_pages/code_of_conduct.md:102]: Is email+slack+form enough? Or should we try to provide a way of submitting verbal reports such as phone of videocall?

File: _2017_pages/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+* Alexandre Andorra, Diversity Chair
+  - andorra.alexandre@gmail.com
+
+### Conflicts of Interest
+
+In the event of any conflict of interest, the committee member will immediately notify
+and recuse themselves if necessary.
+
+If you are concerned about making a report that will be read by any of the above
+individuals, please reach out to [PyMCon's executive committee]() or to one of the members
```
Comment (OriolAbril) [inline _2017_pages/code_of_conduct.md:147]: not sure what to link here nor if it should be linked here.

File: _2017_pages/code_of_conduct.md (hunk 1 above)
Comment (canyon289) [reply _2017_pages/code_of_conduct.md:102]: This is enough, its the standard at pydata in person conferences, and I believe it matches pydata global as well.

File: _2017_pages/code_of_conduct.md (hunk 2 above)
Comment (canyon289) [reply _2017_pages/code_of_conduct.md:147]: The numfocus anonymous. Being honest I don't know exactly know how jekylls markdown rendering works so I'm going to merge this PR as is so I don't mess anything up and create an issue to track this.

File: _2017_pages/code_of_conduct.md (hunk 1 above)
Comment (OriolAbril) [reply _2017_pages/code_of_conduct.md:102]: Okey, then we have to remove this paragraph

File: _2017_pages/code_of_conduct.md (hunk 2 above)
Comment (OriolAbril) [reply _2017_pages/code_of_conduct.md:147]: Numfocus one _can_ be anonymous, but it's not necessary.

File: _2017_pages/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
+---
+layout: 2017_default
```
Comment (AlexAndorra) [inline _2017_pages/code_of_conduct.md:2]: Souldn't it be "2020_default"?

File: _2017_pages/code_of_conduct.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+physical appearance, body size, race, or religion. We do not tolerate harassment of
+participants in any form.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+## Code of Conduct Highlights Specific to PyMCon
+
+We reproduce here certain portions of the NumFOCUS Code of Conduct that are specifically
+relevant to PyMCon. We encourage all PyMCon participants to review the complete Code of
+Conduct.
```
Comment (AlexAndorra) [inline _2017_pages/code_of_conduct.md:51]: This is a duplicate from lines 27-29

File: _2017_pages/code_of_conduct.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+### How to Report
+
+If you believe someone is violating the Code of Conduct, please report this in a timely
+manner. Code of Conduct violations reduce the value of the community for everyone and we
+take them seriously.
+
+You can make a personal or anonymous report by:
+
+* **By Email**: Send a message to any or all Diversity Chairs, which can be reached at
```
Comment (AlexAndorra) [inline _2017_pages/code_of_conduct.md:61]: **~~By~~ Email**: Send a message to any or all Diversity Chairs, ~~which~~ **who** can be reached at

File: _2017_pages/code_of_conduct.md [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+  oriol.abril.pla@gmail.com and andorra.alexandre@gmail.com. Event organizers will respond
+  promptly.
+* **On Slack**: Join the [PyMCon Slack](https://pymcon.slack.com/join/shared_invite/zt-fk9efmvb-SWvT01TOnykgzFmLgn1iAw)
```
Comment (AlexAndorra) [inline _2017_pages/code_of_conduct.md:64]: **~~On~~ Slack**: Join the [PyMCon Slack](https://pymcon.slack.com/join/shared_invite/zt-fk9efmvb-SWvT01TOnykgzFmLgn1iAw) and contact ~~with~~ the Diversity Chairs, Oriol Abril or Alex Andorra.

File: _2017_pages/code_of_conduct.md [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+  and contact with the Diversity Chairs, Oriol Abril or Alex Andorra.
+* **Online Form**: A variant of the NumFOCUS Code of Conduct Report is available for
+  PyMCon [here](https://numfocus.typeform.com/to/kMgloi8t) (your name and contact
+  information are not required).
+
+During PyMCon, we recommend contacting via email or Slack so we can take
+immediate action if necessary. In addition to contacting directly with the Diversity
+committee, we also encourages you to make a report using our reporting form.
```
Comment (AlexAndorra) [inline _2017_pages/code_of_conduct.md:72]: In addition to contacting directly ~~with~~ the Diversity committee, we also encourage~~s~~ you to make a report using our reporting form.

File: _2017_pages/code_of_conduct.md [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+In the event of any conflict of interest, the committee member will immediately notify
+and recuse themselves if necessary.
+
+If you are concerned about making a report that will be read by any of the above
+individuals, please reach out to [PyMCon's executive committee]() or to one of the members
+of the [NumFOCUS Board](https://www.numfocus.org/people#people-directors).
+
+### Appealing a Decision
+
+To appeal a decision, contact Oriol Abril-Pla at oriol.abril.pla@gmail.com with your
```
Comment (AlexAndorra) [inline _2017_pages/code_of_conduct.md:152]: Should my email be here too?

**Review by AlexAndorra — COMMENTED** [review]
Looks good, thanks guys! I found some typos below

File: _2017_pages/code_of_conduct.md (hunk 3 above)
Comment (OriolAbril) [reply _2017_pages/code_of_conduct.md:2]: its either 2016 or 2017, it has to be the name of an existing file

File: _2017_pages/code_of_conduct.md (hunk 4 above)
Comment (OriolAbril) [reply _2017_pages/code_of_conduct.md:51]: yes, but I think it does no harm to repeat twice to read the whole code of conduct

File: _2017_pages/code_of_conduct.md (hunk 8 above)
Comment (OriolAbril) [reply _2017_pages/code_of_conduct.md:152]: if you want to yep, I mimicked numfocus here. My role would only be to forward the appeal to either executive committee or numfocus. As they are challenging our decision, someone else should be the one to evaluate the appeal. Another option is to have them reach Ravin or Thomas directly :thinking:

Comment (AlexAndorra) [reply _2017_pages/code_of_conduct.md:152]: Ah I see. Let's leave it like that then

**Pull request merged**